theforeman.org/pipelines/vars/smart-proxy/3.3.groovy

```diff
@@ -0,0 +1 @@
+git_branch = '3.3-stable'
```

Comment:
These files are really trivial now. I'm still debating what's best. I imagine that we should either find a way to avoid this or expand on this where this file also contains the variables like Ruby versions.

Reply:
can't we just do `git_branch = "{version}-stable"` in some more central place?

Reply:
Any thoughts where? I thought about https://github.com/theforeman/jenkins-jobs/blob/master/theforeman.org/pipelines/vars/foreman/3.5.groovy but wonder if it makes sense.

Reply:
foreman follows the same branching structure, so if you also want to replace the foreman stable_X_Y jobs (and I guess we do), that's a good place?

Reply:
> if you also want to replace the foreman stable_X_Y jobs

That is indeed the intent.